tests_metricflow/integration/test_configured_cases.py

```diff
@@ -261,8 +261,6 @@ def test_case(
         if date_part or grain:
             if date_part:
                 kwargs["date_part"] = DatePart(date_part)
-            if grain:
```

Comment:
how did this not break anything 🤔

Reply:
This is the fix for a change to TimeDimensionParameter above. 

We no longer need to convert the string input to a TimeGranularity in the TimeDimensionParameter initializer - it accepts string values - so without this change we were passing an incorrectly typed value in for that argument and weird runtime failures were cropping up.